pyasn1_modules/rfc3739.py

```diff
@@ -158,6 +160,12 @@ class SemanticsInformation(univ.Sequence):
         namedtype.OptionalNamedType('nameRegistrationAuthorities',
             NameRegistrationAuthorities())
     )
+    subtypeSpec = constraint.ConstraintsUnion(
```

Comment:
Would it make sense to have a unit test against a broken data structure i.e. w/o any of these components?

Reply:
I can work on a unit test to test for a failure.

Reply:
Would you like to work on that within this PR or some other?

Reply:
I will add it to the PR shortly

Reply:
The updated test_rfc3739.py has been added.